dts/binding-template.yaml

```diff
@@ -158,6 +168,31 @@ properties:
         required: true
         const: 1
 
+    int-with-default:
+        type: int
+        category: optional
```

Comment:
Examples need updating for `required` instead of `category`.

Reply:
Will fix.

Reply:
Fixed